### ISSUE TITLE
Possible that clients are changing branches?

### ISSUE THREAD

**Issue by jtg567** [mozilla/normandy#958]
Analyzing some crash pings for a pref-flip I discovered a subset of clients with pings for both branches of the study that seem to alternate haphazardly over time (rather than changing once over time) 

CTRL+F for `weirdos.take(weirdos.count())` to see at the following gist
https://gist.github.com/jtg567/6c288bc8ded498ffdd673aec3df1ff45

**Comment by mythmon** [comment]
I've got a couple ideas here:

1. There is a bug in our code that makes us forget or otherwise go back on which branch we decided
2. Some users have "privacy" settings that are even clearing indexeddb for add-ons. We don't have any explicit checks against this right now.
3. Firefox profiles are being copied after the Telemetry id has been created, but before this recipe ran. That would cause them to be sampled separately by Normandy. If it is possible to check the Normandy id, separate from the Telemetry id, you'd be able to confirm this.
